src/ebfm/INIT.py

```diff
@@ -338,7 +341,27 @@ def init_grid(grid, io, args: Namespace):
         grid["slope_gamma"][(grid["slope_x"] > 0) & (grid["slope_y"] == 0)] = np.pi / 2
         grid["slope_gamma"][(grid["slope_x"] < 0) & (grid["slope_y"] == 0)] = -np.pi / 2
         grid["slope_gamma"] = -grid["slope_gamma"]
+    elif grid["input_type"] is GridInputType.ELMER_XIOS:
```

Comment:
Does this work?

It's generally a good idea to make sure that at least one of the cases in this chain of `if-elif`-statements is entered. I always append a final `else: raise Exception(f"Cannot handle unexpected {grid["input_type"]=}")`. This will make sure that the program exits.